docs/installation/hosted-ce.md

```diff
@@ -0,0 +1,3 @@
+Installing a Hosted CE
```

Comment:
Is this a stub?

Reply:
Yeah, it's not actually in `mkdocs.yml` yet